pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py

```diff
@@ -12,27 +12,98 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 import os
+from pprint import pprint
+from typing import Iterable, Union
+from copy import deepcopy
+from collections import ChainMap
 import torch
 from pytorch_lightning.core import memory
 from pytorch_lightning.loggers import TensorBoardLogger, LoggerCollection
 from pytorch_lightning.utilities import flatten_dict
 from pytorch_lightning.utilities.model_utils import is_overridden
 from pytorch_lightning.core.step_result import EvalResult, Result
 from pytorch_lightning.utilities.exceptions import MisconfigurationException
-from pprint import pprint
-from typing import Iterable
-from copy import deepcopy
-from collections import ChainMap
+from pytorch_lightning.trainer.connectors.logger_connector.callback_hook_validator import CallbackHookNameValidator
+from pytorch_lightning.trainer.connectors.logger_connector.epoch_result_store import (
+    EpochResultStore,
+    LoggerStages
+)
 
 
 class LoggerConnector:
 
+    __lookup_stages = {"1": "test", "0": "validation", "True": "test", "False": "validation"}
+
     def __init__(self, trainer):
         self.trainer = trainer
         self.callback_metrics = {}
         self.logged_metrics = {}
         self.progress_bar_metrics = {}
         self.eval_loop_results = []
+        self._stages = sorted([s.value for s in LoggerStages])
```

Comment:
Why do we use the value of the enum rather than the enum itself?

Reply:
Because I am not a huge fan of Enum and a dictionary was easier to manipulate.